js/plugin/popups.ts

```diff
@@ -55,7 +55,6 @@ export const PopupMixin = (SuperClass) => {
           tag != "" ?
             `browser-mod-popup-${tag}` :
             "browser-mod-popup";
-            "browser-mod-popup";
```

Comment:
Not sure why linter did not pick this up. Was not included in any transpiled code.

Reply:
It's valid. Just doesn't do anything.